pkg/container/nulls/nulls.go

```diff
@@ -64,20 +67,28 @@ func Any(n *Nulls) bool {
 	return !n.Np.IsEmpty()
 }
 
+// Any returns true if any bit in the Nulls is set, otherwise it will return false.
```

Comment:
Look at this comment -- exactly the same as Any, what did I miss?   Is preciseAny the same as Any?

Other than that, this API is broken -- how does a caller know which one to call?   I think you need only one method.